app/main.py

```diff
@@ -2,24 +2,84 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = name
+
+    def __get__(self,
+                instance: "SlideLimitationValidator",
+                owner: type
+                ) -> tuple:
+        return getattr(instance, self.name)
+
+    def __set__(self,
+                instance: "SlideLimitationValidator",
```

Comment:
Instead of `SlideLimitationValidator` indicate `Any` in the type annotation here and in the `__get__` method to not limit `IntegerRange` applicability